Training_25/Program.cs

```diff
@@ -3,11 +3,32 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program takes a string and returns the reversed string
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
+
 namespace Training_25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      Write ("Enter a string: ");
+      WriteLine ($"Reversed string: {StringReverser (ReadLine ())}");
+   }
+
+   // Returns the reversed string for the entered string by preserving the order of casing and spaces
+   static string StringReverser (string? input) {
+      if (string.IsNullOrWhiteSpace (input)) return "";
+      char[] result = new char[input.Length];
+      int j = input.Length - 1;
+      for (int i = 0; i < input.Length; i++) {

```

Comment:
Can initialize variable j in the for loop itself.
```suggestion
      for (int i = 0, j = input.Length - 1; i < input.Length; i++, j--) {

```